### PR TITLE
Add shakespeare sample for BigQuery.

### PR DESCRIPTION
**Pull request by tswast**
For using veneer at https://cloud.google.com/bigquery/create-simple-app-api

@lesv PTAL

### CODE REVIEW

File: bigquery/cloud-client/README.md [hunk 1]
```diff
@@ -31,3 +31,11 @@ You can then run a given `ClassName` via:
     mvn exec:java -Dexec.mainClass=com.example.bigquery.SyncQuerySample \
         -Dquery='SELECT corpus FROM `publicdata.samples.shakespeare` GROUP BY corpus;' \
         -DuseLegacySql=false
+
+### Running the simple app example
+
+To run the example from the [simple app example
+documentation](https://cloud.google.com/bigquery/create-simple-app-api):
+
+    mvn exec:java -Dexec.mainClass=com.example.bigquery.SimpleApp
```
Comment (lesv) [inline bigquery/cloud-client/README.md]: Can't you add a manifest line in your pom.xml so you don't need this?

File: bigquery/cloud-client/src/main/java/com/example/bigquery/SimpleApp.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    // [END create_client]
+    // [START run_query]
+    QueryRequest queryRequest =
+        QueryRequest
+            .builder(
+                "SELECT "
+                    + "APPROX_TOP_COUNT(corpus, 10) as title, "
+                    + "COUNT(*) as unique_words "
+                    + "FROM `publicdata.samples.shakespeare`;")
+            // Use standard SQL syntax for queries.
```
Comment (lesv) [inline bigquery/cloud-client/src/main/java/com/example/bigquery/SimpleApp.java]: SQL or GQL?

File: bigquery/cloud-client/README.md (hunk 1 above)
Comment (tswast) [inline bigquery/cloud-client/README.md]: Do we have an example of that?

File: bigquery/cloud-client/src/main/java/com/example/bigquery/SimpleApp.java (hunk 2 above)
Comment (tswast) [inline bigquery/cloud-client/src/main/java/com/example/bigquery/SimpleApp.java]: It's called standard SQL publicly. https://cloud.google.com/bigquery/sql-reference/enabling-standard-sql

**Pull request merged**